app/views/sample_types/_sample_attribute_form.html.erb

```diff
@@ -52,7 +52,7 @@
   </td>
   <td>
     <%= select_tag "#{field_name_prefix}[sample_attribute_type_id]",
-                   options_for_select(SampleAttributeType.all.sort_by(&:title).sort_by { |t| t.default? ? 0 : 1 }.map { |t| [t.title, t.id,{'data-is-cv'=>t.controlled_vocab?,'data-is-ontology'=>t.ontology?, 'data-is-seek-sample'=>t.seek_sample? || t.seek_sample_multi? }] }, attribute_type_id),
+                   options_for_select(SampleAttributeType.all.reject{|t| t.base_type == 'List'}.sort_by(&:title).sort_by { |t| t.default? ? 0 : 1 }.map { |t| [t.title, t.id,{'data-is-cv'=>t.controlled_vocab?,'data-is-ontology'=>t.ontology?, 'data-is-seek-sample'=>t.seek_sample? || t.seek_sample_multi? }] }, attribute_type_id),
```

Comment:
The change of this line is to hide the "List" type from the select options of sample attribute type.
because we don't provide the UI to create a "List" type yet. 
<img width="734" alt="Screenshot 2023-01-17 at 11 52 27" src="https://user-images.githubusercontent.com/42063448/212880636-9f4670ec-e79b-4552-ba48-35f360e87864.png">